webpack.config.js

```diff
@@ -0,0 +1,37 @@
+const HtmlWebpackPlugin = require("html-webpack-plugin");
```

Comment:
Webpack adds some optimisation for production builds so you should have created separate config files for different environments.